content/applications/inventory_and_mrp/inventory/shipping/setup/sendcloud_shipping.rst

```diff
@@ -0,0 +1,189 @@
+
+- :guilabel:`Shipping Method`: type `Sendcloud DPD`.
+- :guilabel:`Provider`: select :guilabel:`Sendcloud` from the drop-down menu.
+- :guilabel:`Delivery Product`: set the product that was configured for this shipping method or
+  create a new product.
+- In the :guilabel:`SendCloud Configuration` tab, enter the :guilabel:`Sendcloud Public Key`.
+- In the :guilabel:`SendCloud Configuration` tab, enter the :guilabel:`Sendcloud Secret Key`.
+- Manually :guilabel:`Save` the form by clicking the cloud icon next to the :guilabel:`Shipping
+  Methods / New` breadcrumbs.
+
+After configuring and saving the form, follow these steps to load the shipping products:
+
+- In the :guilabel:`SendCloud Configuration` tab of the :guilabel:`New Shipping Method` form, click
+  on the :guilabel:`Load your SendCloud shipping products` link.
+- Select the shipping products the company would like to use for deliveries and returns.
+- Click :guilabel:`Select`.
+
+For example:
+
+.. image:: sendcloud_shipping/sendcloud-example.png
+   :align: center
+   :alt: Example of shipping products configured in Odoo.
+
+.. tip::
+   Sendcloud does not provide test keys when a company tests the sending of a package in Odoo. This
+   means if a package is created, the configured Sendcloud account will be charged, unless the
+   associated package is canceled within 24 hours of creation.
+
+   Odoo has built an extra layer of protection into test environments. If the shipping method is
+   used to create a label, then the labels are immediately canceled after the creation. This occurs
+   automatically.
+
+Generate a label with Sendcloud
+-------------------------------
+
+When creating a quotation in Odoo, add shipping and a :guilabel:`Sendcloud shipping product`. Then,
+:guilabel:`Validate` the delivery. Shipping label documents will be automatically generated in the
+chatter.
+
+The following should be included in the shipping label documents:
+#. :guilabel:`Shipping label(s)` (depending on the number of packages).
+#. :guilabel:`Return label(s)` (if the Sendcloud connector is configured for returns).
+#. :guilabel:`Customs document(s)` (should the destination country require them).
+
+Additionally, the tracking number is also available.
+
+.. note::
+   If needed, shipping can be canceled by navigating to the :guilabel:`Additional info` tab of the
+   delivery order.
+
+FAQ
+===
+
+Shipment is too heavy
+---------------------
+
+If the shipment is too heavy for the Sendcloud service that is configured, then the weight is split
+to simulate multiple packages. Products will need to be put in different :guilabel:`Packages` to
+:guilabel:`Validate` the transfer and generate labels.
+
+:guilabel:`Rules` can also be set up in Sendcloud to use other shipping methods when the weight is
+too heavy. However, it should be noted that these rules will not apply on the price calculation in
+the :guilabel:`Sales Order`.
```

Comment:
```suggestion
the sales order.
```

Removed the `:guilabel:` after thinking about it, this is more referring to a `sales order` in the general sense, rather than a specific `sales order` that the user needs to interact with in this paragraph.